.dockerignore

```
@@ -0,0 +1,19 @@
+.dockerignore
+.git
+.gitignore
+Dockerfile
+Jenkinsfile
+Procfile
+README.md
+CONTRIBUTING.md
+adr
+carrierwave-tmp
+coverage
+docs
+features
+log
+node_modules
+spec
+test
+tmp
+uploads
```

```suggestion
coverage
docs
log
node_modules
spec
tmp
```

It looks like you've picked up a few Whitehall specific quirks in this list, so just I've just suggested reducing it to actual paths this app uses.

---

Ah good shout - thanks